server/endpoint/health/searcher/response.go

```diff
@@ -1,5 +1,48 @@
 package searcher
 
+// GeneralStatus holds data about the overall health/status of a cluster
+type GeneralStatus struct {
+	Health       string `json:"health"`
+	Creating     bool   `json:"creating"`
+	Upgrading    bool   `json:"upgrading"`
+	Deleting     bool   `json:"deleting"`
+	Normal       bool   `json:"normal"`
+	NodeCount    int    `json:"nodeCount"`
```

Comment:
```suggestion
	NodeCount    int    `json:"node_count"`
```
We usually camel case like this in our other APIs when json is involved - I am going to be honest though, no idea why.

Reply:
Hmm, a little Python flair in our Go? Fixed.